sdk/core/azure_core/Cargo.toml

```diff
@@ -49,7 +49,7 @@ rustc_version = { workspace = true }
 
 [dev-dependencies]
 tracing-subscriber = { workspace = true }
-tokio = { workspace = true }
+tokio = { workspace = true, features = ["rt-multi-thread"] }
```

Comment:
We don't want to do this in every crate. Instead, leave it as it was but in your crates do:

```toml
tokio = { workspace = true, default-features = false, features = ["whatever"] }
```

you may have to add `default-features = false` to the root `Cargo.toml`, but I already did to some other ones and list out what features I want by default. That's fine.